cli/cmd/generate.go

```diff
@@ -54,7 +54,7 @@ var (
 	// aws command is used to generate TF code for aws
 	generateAwsTfCommand = &cobra.Command{
 		Use:   "aws",
-		Short: "Genereate terraform code for deploying into a new AWS environment",
+		Short: "Generate and/or execute Terraform code for AWS deployment",
```

Comment:
Would be good to get input from others but I prefer this to the original. Although I would personally replace `deployment` with `integration`

Reply:
done.